endpoints/openrtb2/video_auction.go

```diff
@@ -203,8 +203,9 @@ func (deps *endpointDeps) VideoAuctionEndpoint(w http.ResponseWriter, r *http.Re
 	// Populate any "missing" OpenRTB fields with info from other sources, (e.g. HTTP request headers).
 	deps.setFieldsImplicitly(r, bidReq) // move after merge
 
+	// here be the problem!
 	errL = deps.validateRequest(bidReq)
-	if len(errL) > 0 {
+	if errortypes.ContainsFatalError(errL) {
```

Comment:
This is the main fix. I added similar filters to AMP and Auction, but forgot about Video.